draft-salowey-tls-rfc8447bis.md

```diff
@@ -102,32 +102,34 @@ to also refer to this document.
 
 # Adding Recommended Column
 
-The instructions in this document add a Recommended column to many of
-the TLS registries to indicate parameters that are generally recommended
-for implementations to support, parameters that are not recommended for
-specific reasons, and parameters that have not been evauated.
-
-- Parameters that are Recommended are marked with a 'Y'.
-- Parameters that are Not Recommended are maked with a 'N' and include
-  a reference to the document and section which describes the limitation
-  of the parameter.
-- Parameters that are Unevaluated are unmarked with a ' '.
-
-Adding a Recommended parameter to a registry or updating a parameter to
-Recommended status requires standards action.  Not all parameters defined
-in standards track documents need to be marked as Recommended.  Changing
-the recommended status of a standards track parameter requires standards action.
-
-If an item is not marked as Not Recommended it does not necessarily mean
-that it is flawed in all cases.  A parameter that is not recommended may
-have limited applicability, or may be intended only for specific use cases
-as indicated in the referenced text.
-
-If an item is marked as Unevaluated it indicates that either the item has not
-been through the IETF consensus process and it may have limited applicability,
-or may be intended only for specific use cases.
-
-[Note: the registries in the rest of the document will need to have the recommended column updated appropriately]
+The instructions in this document update the Recommended column,
+originally added in {{?!RFC8447}} to add a third value, "D",
+indicating that a value is "Discouraged". The permitted values
+are:
+
+- Y: Indicates that the IETF has consensus that the
+  item is RECOMMENDED.
+- N: Indicates that the item has not been evaluated by
+  the IETF.
+- D: Indicates that the item is discouraged and SHOULD
+  NOT or MUST NOT be used.
+
+Setting the Recommended item to "Y" or "D" or changing a
+item whose current value is "Y" or "D" requires standards action.
+Not all items defined in standards track documents need to be
+marked as Recommended. Changing the Recommended status of a standards
+track item requires standards action.
+
+If an item is not marked as "Recommended" (i.e., "N"), it does not
+necessarily mean that it is flawed; rather, it indicates that the
+item either has not been through the IETF consensus process, has
+limited applicability, or is intended only for specific use cases.
+Note that items which have IETF consensus and which are believed
+to be generally safe MAY be marked "Y" even if they are only
+intended for some specific application, such as WebRTC or IoT.
```

Comment:
This is where I think we run into issues.

This is trying to pre-decide something. It's almost a disclaimer on "Y".  And it might be better to place that disclaimer on the definition of "Y".  Maybe instead.

> An item with a Recommended value of "Y" indicates that the IETF has consensus to recommend that item. This only means that the associated mechanism is fit for the purpose for which it was defined. Careful reading of the documentation for the mechanism is necessary to understand the applicability of that mechanism.  The IETF could recommend mechanisms that have limited applicability, but will provide applicability statements that describe any limitations of the mechanism or necessary constraints on its use.

> An item with a Recommended value of "N" indicates that the IETF has made no statement about the suitability of the associated mechanism. This does not necessarily mean that the mechanism is flawed, only that no consensus exists. The IETF might have consensus to leave an items marked as "N" on the basis of it having limited applicability or usage constraints.

> An item with a Recommended value of "D" indicates that the IETF has consensus to discourage use of the associated mechanism. This marking could be used to identify mechanisms that might result in problems if they are used, such as a weak cryptographic algorithm or a mechanism that might cause interoperability problems in deployment.